packages/full-stack-tests/src/hierarchy-builder/Localization.test.ts

```diff
@@ -0,0 +1,32 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import { expect } from "chai";
+import { IModelApp } from "@itwin/core-frontend";
+import { createLocalizationFunction } from "@itwin/presentation-core-interop";
+import { setLocalizationFunction } from "@itwin/presentation-hierarchy-builder";
+import { translate } from "@itwin/presentation-hierarchy-builder/lib/cjs/hierarchy-builder/Localization";
+import { initialize, terminate } from "../IntegrationTests";
+
+describe("Stateless hierarchy builder", () => {
+  describe("Localization", () => {
+    describe("translate", () => {
```

Comment:
I don't think we need this "translate" describe.

Reply:
Removed